customers/views.py

```diff
@@ -25,17 +26,8 @@ def create():
             email=email,
             dob=dob,
         )
-        db.session.add(newCustomer)
-        db.session.commit()
-        return {
-            "id": f"{newCustomer.id}",
-            "username": f"{userName}",
-            "first_name": f"{firstName}",
-            "last_name": f"{lastName}",
-            "email": f"{email}",
-            "dob": f"{dob}",
-            "date_created": f"{newCustomer.date_created}",
-        }, 200
+        newCustomer.save()
+        return (customerSchema.jsonify(newCustomer)), 200
```

Comment:
double check the response for this, I'm wondering why `customerSchema.jsonify(newCustomer)` is wrapped in parens